admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -1380,7 +1380,11 @@ protected function _default_tickets_update(EE_Event $event, $data)
             $datetime->_add_relation_to($ticket, 'Ticket');
             $saved_tickets[ $ticket->ID() ] = $ticket;
             // add prices to ticket
-            $this->_add_prices_to_ticket($data['edit_prices'][ $row ], $ticket, $update_prices);
+            $prices_data = (
+                isset($data['edit_prices'][ $row ])
+                && is_array($data['edit_prices'][ $row ])
+            ) ? $data['edit_prices'][ $row ] : [];
```

Comment:
let's format this like:

```php
            $prices_data = isset($data['edit_prices'][ $row ]) && is_array($data['edit_prices'][ $row ])
            	? $data['edit_prices'][ $row ] 
            	: [];
```

or if that conditional is too long, then something like:

```php
            $prices_data = isset($data['edit_prices'][ $row ]) 
				&& is_array($data['edit_prices'][ $row ])
            		? $data['edit_prices'][ $row ] 
            		: [];
```

if you have composer installed you can run the codesniffer linter:

```
// run config first
composer config-eventespressocs

// to check for errors
composer lint:skip-warnings

// and attempt to auto-fix
composer lint:fix
```